src/web_app_skeleton/script/setup.ecr

```diff
@@ -40,6 +57,9 @@ if [ ! -f ".env" ]; then
   printf "✔ Done\n" | indent
 fi
 
+printf "\n▸ Checking that postgres is installed and running\n"
```

Comment:
Maybe do what we did on line 57 and print  done after the check so it is clear it succeeded?